.readthedocs.yaml

```diff
@@ -8,7 +8,7 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.11"
+    python: "3.12"
```

Comment:
I have merged this separately - this has no effect on the images (and probably no effect on the docs, so I use the version suggested by the RTD in the link above).